src/utilities/kubectlUtils.ts

```diff
@@ -53,12 +58,13 @@ export async function annotateChildPods(
    kubectl: Kubectl,
    resourceType: string,
    resourceName: string,
+   namespace: string | undefined,
```

Comment:
is there a reason why this isn't just `namespace?: string`

Reply:
optional params need to be after other ones.